src/courses.json

```diff
@@ -29,8 +29,9 @@
   "Einführung in die Informatik ": "INFO",
   "Praktikum: Grundlagen der Programmierung": "PGP",
   "Einführung in die Rechnerarchitektur": "ERA",
+  "Grundlagenpraktikum: Rechnerarchitektur": "GRA",
```

Comment:
Is this the ERA-Praktikum in the second semester? If yes this should be named ERA-P imo

Reply:
Yeah, that is the ERA-Praktikum.
I just wanted to keep it aligned with the official website of the course: https://gra.caps.in.tum.de/login

But ERA-P is fine for me aswell. You can decide :)